plugins/by-name/faust/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, pkgs, ... }:
+let
+  globalPrefix = "faust";
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "faust";
+  packPathName = "faust-nvim";
+  package = "faust-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  # Configuration is done through globals vim.g.faust*
```

Comment:
I'm confused. Why are we using `mkNeovimPlugin` and then configuring it to behave like `mkVimPlugin`?

Reply:
Hm. I missed that this plugin has some lua-based setup _in addition_ to globals-based settings.

I think this approach is fine, then.